src/ethereum_test_vm/bytecode.py

```diff
@@ -117,6 +117,14 @@ def __eq__(self, other):
         - NotImplementedError: if the comparison is not between an Bytecode
             or a bytes object.
         """
+        if isinstance(other, Bytecode):
+            return (
+                bytes(self) == bytes(other)
```

Comment:
i think this must be enough. because pushed popped is our own construct. 
I use opcode generation from int number. and I don't provide the poped or pushed values.